src/IonSpan.ts

```diff
@@ -63,7 +63,7 @@ export abstract class Span {
 }
 
 export class StringSpan extends Span {
-  private _src : string;
+  _src : string;
```

Comment:
yes, however since its a string in the heap I'm not sure I see the difference since this is an internal call on a private attribute of the reader:
export class TextReader implements Reader {
  private _parser: ParserTextRaw;
...}

Reply:
removing `private` will allow read **and write** to the field from anywhere--a more drastic change to the code. If the reason for removing `private` was only to read the value, a getter fits the need and does not open up the door for mutating the variable from anywhere.

Reply:
the parser is private already

Reply:
This is `StringSpan` which is exported.

Reply:
oh good point